examples/example-enterprise-deployment.yaml

```diff
@@ -0,0 +1,287 @@
+# 01 - Custom resources
```

Comment:
We don't want to need to maintain another set of manifests. We should add an option to the helm chart for setting the default image that the operator uses and then get users to install via helm.